browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -405,34 +403,32 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 		// Early exit. If there is no need to update the size, return here.
 		if (oldMapSize[0] === newMapSize[0] && oldMapSize[1] === newMapSize[1])
 			return false;
+		else {
```

Comment:
Not needed alignment change, if above just returns -> code below will not be executed.
So we can avoid `else` and additional margin

Reply:
Yes, this is a matter of style.